.github/workflows/continuous-integration.yml

```
@@ -78,7 +82,7 @@ jobs:
           python -m pip install jupyter-fs || true
       - name: Install markdown-it-py
         if: ${{ matrix.markdown-it-py-version }}
-        run: python -m pip install "markdown-it-py~=${{ matrix.markdown-it-py-version }}"
+        run: python -m pip install markdown-it-py~=${{ matrix.markdown-it-py-version }}
```

```suggestion
        run: python -m pip install markdown-it-py${{ matrix.markdown-it-py-version }}
```
I usually embed the sign in it as well.
https://github.com/executablebooks/sphinx-design/pull/131/files